src/me/ryanhamshire/PopulationDensity/TeleportPlayerTask.java

```diff
@@ -15,120 +15,92 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
- 
- package me.ryanhamshire.PopulationDensity;
 
-import java.util.ArrayList;
-import java.util.List;
+package me.ryanhamshire.PopulationDensity;
 
 import org.bukkit.Location;
 import org.bukkit.Sound;
-import org.bukkit.entity.AbstractHorse;
-import org.bukkit.entity.AnimalTamer;
-import org.bukkit.entity.Animals;
-import org.bukkit.entity.Entity;
-import org.bukkit.entity.EntityType;
-import org.bukkit.entity.LivingEntity;
-import org.bukkit.entity.Ocelot;
-import org.bukkit.entity.Player;
-import org.bukkit.entity.Tameable;
-import org.bukkit.entity.Wolf;
+import org.bukkit.entity.*;
```

Comment:
Please change your IDE settings to _not_ do this. (and try to undo accordingly, thanks).